src/config/configProcessors.js

```diff
@@ -109,7 +109,35 @@ function configProcessors(helper, eventProcessor, eventOptions,
     // A failure occurred, return now to prevent the page from crashing.
     return;
   }
+  registerEventAndSet_(helper, processor, eventOptions, storage);
+}
 
+/**
+ * Register the set and event commands on the helper with the given
+ * processor/storage pair.
+ *
+ * @private
+ * @param {!DataLayerHelper} helper
+ * @param {!EventProcessor} processor
+ * @param {!Object<string, *>} eventOptions This will be used in a future PR
+ * @param {!StorageInterface} storage
+ */
+function registerEventAndSet_(helper, processor, eventOptions, storage) {
+  // Check if a given key/value pair should be persisted in storage, and
+  // if so, save it.
+  function processSet(key, value, secondsToLive = undefined) {
+    if (secondsToLive === undefined) {
+      secondsToLive = processor.persistTime(key, value);
+    }
+    // Don't save if the time to live is 0.
+    if (secondsToLive) {
```

Comment:
Is this only for checking against secondsToLive === 0? What other values could processor.persistTime return?

Reply:
According to the [contract](https://github.com/googleinterns/measurement-library/blob/master/src/eventProcessor/EventProcessorInterface.js), it is either -1 or a number greater than or equal to 0 (Including infinity). The undefined default value is probably confusing, I'm following [the tagging guideline](https://github.com/googleinterns/measurement-library/blob/master/src/eventProcessor/EventProcessorInterface.js) but I every time I've followed this guideline so far, someone has posted a comment on my PR with some amount of confusion, so it may be a guideline that is better not to follow...

Reply:
I added jsdoc to this function because it may help future readers understand the possible values of seconds to live

Reply:
Ah, this recommendation doesn't actually apply since I need an explicit undefined check (the recommendation was to prefer a falsy check), rewriting it actually saves 1 byte.